templates/secret_santa/committee_mail.txt

```diff
@@ -1,2 +1,4 @@
 Secret santa e-post har blitt sendt ut.
-Det er lagret en kopi av alle til-fra par om det blir behov for å sjekke dette.
\ No newline at end of file
+
+Det er lagret en liste med alle til-fra par.
+Ta kontakt med DotKom om om det blir behov for å sjekke denne.
```

Comment:
It's not DotKom, nor Dotkom, it's just dotkom.

Reply:
#altvarbedrefør